bitwarden_license/src/Commercial.Core/SecretsManager/Queries/Projects/MaxProjectsQuery.cs

```diff
@@ -29,15 +29,15 @@ public async Task<(short? max, bool? overMax)> GetByOrgIdAsync(Guid organization
         }
 
         var plan = StaticStore.GetPlan(org.PlanType);
-        if (plan == null)
+        if (plan?.SecretsManager == null)
         {
             throw new BadRequestException("Existing plan not found.");
         }
 
         if (plan.Type == PlanType.Free)
         {
             var projects = await _projectRepository.GetProjectCountByOrganizationIdAsync(organizationId);
-            return projects >= plan.SecretsManager.MaxProjects ? ((short?)plan.SecretsManager.MaxProjects, true) : ((short?)plan.SecretsManager.MaxProjects, false);
+            return ((short? max, bool? overMax))(projects + projectsToAdd > plan.SecretsManager.MaxProjects ? (plan.SecretsManager.MaxProjects, true) : (plan.SecretsManager.MaxProjects, false));
```

Comment:
It looks like you've changed the logic here, but I agree with you. This looks like a bug. @bitwarden/team-secrets-manager-dev can confirm though.